api/src/config.rs

```diff
@@ -938,6 +942,9 @@ pub struct MirrorConfig {
     /// Maximum number of failures before marking a mirror as unusable.
     #[serde(default = "default_failure_limit")]
     pub failure_limit: u8,
+    /// Elapsed time to pause mirror health check when the request is inactive, in seconds.
+    #[serde(default = "default_check_pause_elapsed")]
+    pub check_pause_elapsed: u64,
```

Comment:
We have `health_check_interval`, so maybe name it `health_check_pause_elapsed` ?

Reply:
Ok

Reply:
Fixed, PTAL :) @imeoer